core/src/migrations/000101-removeLogsTable.ts

```diff
@@ -0,0 +1,11 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (queryInterface: Sequelize.QueryInterface) => {
+    await queryInterface.dropTable("logs");
+  },
+
+  down: async () => {
+    throw new Error("irreversible migration");
```

Comment:
Should this throw, or just log? Not sure if we have a `run all down` type command.

Reply:
We've got a couple of these already.  They mainly appear when you can't recover the data you lost.  I'm on the fence about this one because the log table was always meant to be ephemeral, so we could possibly just re-create an empty table and that could be OK